tests/test_adjointA.py

```diff
@@ -17,16 +18,16 @@ def test_acoustic(dimensions, time_order, space_order, fix_dim):
     srca = PointSource(name='srca', ntime=solver.source.nt,
                        coordinates=solver.source.coordinates.data)
 
-    if fix_dim:
-        time.size = solver.source.nt
-    else:
-        time.size = None
+    # Set fixed ("baked-in") time dimension if requested
+    time.size = solver.source.nt if fix_dim else None
+
     # Run forward and adjoint operators
     rec, _, _ = solver.forward(save=False)
     solver.adjoint(rec=rec, srca=srca)
 
     # Actual adjoint test
     term1 = np.dot(srca.data.reshape(-1), solver.source.data)
     term2 = linalg.norm(rec.data) ** 2
-    print(term1, term2, ("%12.12f") % (term1 - term2), term1 / term2)
+    info("L1 norm: %f, L2 norm: %f, L1-L2: %12.12f, L1/L2: %f"
+         % (term1, term2, term1 - term2, term1 / term2))
```

Comment:
Well remove if you want, let's just hope we won;t go back to what happen last time we did it, which is, all pass with BS values.